sys/include/net/gcoap.h

```diff
@@ -639,10 +639,12 @@ extern "C" {
 #define GCOAP_VFS_EXTRA_STACKSIZE   (0)
 #endif
 
+#ifndef GCOAP_STACK_SIZE
 #define GCOAP_STACK_SIZE (THREAD_STACKSIZE_DEFAULT + DEBUG_EXTRA_STACKSIZE \
                           + sizeof(coap_pkt_t) + GCOAP_DTLS_EXTRA_STACKSIZE \
                           + GCOAP_VFS_EXTRA_STACKSIZE)
 #endif
+#endif
```

Comment:
Hum, so now `GCOAP_STACK_SIZE` is nested within `#ifndef GCOAP_DTLS_EXTRA_STACKSIZE`, looks suspicious. If intended, would need an explanatory comment and maybe a closing `#endif` comment

Reply:
uh, good catch